src/appmixer/notion/auth.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+module.exports = {
+    type: 'oauth2',
+
+    definition: () => {
+        const apiVersion = '2022-06-28';
```

Comment:
`apiversion` is  on two places, on here and API_VERSION in the libs.js. Please consider putting in the config (I'd be config property in the backoffice then)